spec/requests/api/v1/targets/create_spec.rb

```diff
@@ -1,9 +1,13 @@
 describe 'POST api/v1/targets', type: :request do
   let(:user) { create(:user) }
+  let(:user2) { create(:user) }
   let(:topic)           { create(:topic) }
   let(:target)          { Target.last }
 
   describe 'POST create' do
+    before do
+      create(:target, user_id: user2.id, topic_id: topic.id)
+    end
```

Comment:
```suggestion
    let!(:target) { create(:target, user_id: user2.id, topic_id: topic.id) }
```
I recommend using `let` variables to create data instead of using `before` blocks. Take a look at this: https://github.com/rootstrap/tech-guides/blob/master/ruby/rspec/tutorial/let-vs-let.md

Reply:
Great, advice taken! Thanks